packages/list/index.tsx

```diff
@@ -394,6 +384,23 @@ export default class List extends React.Component<ListProps, ListState> {
       tag: Tag,
       ...otherProps
     } = this.props;
+
+    const getListProps = (checkboxList?: boolean, radioList?: boolean) => ({
```

Comment:
I think this needs to be a class property, just like before. Defining this on render every time will reset the `memoize-one` cache 😔

Reply:
ah! Thanks for catching that. I will update. Been working on the select. But will get back to this.